mbs_results/utilities/mapping_validation.py

```diff
@@ -0,0 +1,97 @@
+import warnings
+
+import pandas as pd
+
+
+def wrap_mapping_validations(df: pd.DataFrame, mapping_folder: str):
+    """
+    wrapper to loop over the specified mapping files in file_and_column_names dict.
+    Calls mapping_validation for the specified files
+    Probably could move the files and column names to the config if needed.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        input data to test against mapping files
+    mapping_folder : str
+        folder where the mapping files can be located.
+    """
+    files_and_column_names = {
+        "sic_sut_mapping.csv": {
+            "df_column_name": "sic_5_digit",
+            "mapping_file_col_name": "sic",
+        },
+        "classification_sic_mapping.csv": {
+            "df_column_name": "sic_5_digit",
+            "mapping_file_col_name": "sic_5_digit",
+        },
+        "sic_domain_mapping.csv": {
+            "df_column_name": "sic_5_digit",
+            "mapping_file_col_name": "sic_5_digit",
+        },
+    }
+
+    for i in files_and_column_names:
+        mapping_path = mapping_folder + i
```

Comment:
Minor comment: worth having something in the docstring to say the mapping_folder needs to have a '/' at the end for a readable file path? (or validation?)

Reply:
Updated the doc string to be more explicit about this, nice spot!